src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java

```diff
@@ -43,9 +45,31 @@ private Course requestToCourse(CreateCourseRequest request){
     }
 
 
-    public List<Course> getCourses() {
+    public List<CourseResponse> getCourses() {
 
-        return repository.findAll();
+        List<Course> unchanged = repository.findAll();
+        List<CourseResponse> newList = new ArrayList<>();
+
+        for (Course c:unchanged) {
+            CourseResponse changed = new CourseResponse();
+
+            changed.set_id(c.get_id());
+            int indicator = c.getCategory();
```

Comment:
It is not necessary to create a variable, you can evaluate the category field directly in the IF

Reply:
done